backend/src/controllers/squad.js

```diff
@@ -0,0 +1,101 @@
+exports.getSquad = async (req, res, next) => {  
+  try {
+    // Fetch squad from database
+    const squad = await req.context.models.Squad.findByPk(req.params.squadId);
+    const squadData = squad.dataValues;
+  
+    // Return squad
+    res.status(201);
+    res.send(squadData);
+  } catch (err) {
+    console.log(err)
+    res.status(404);
+    res.send({ error: "Could not find squad." });
+  }
+}
+
+exports.getSquads = async (req, res, next) => {
+  try {
+    // Fetch squads from database
+    const squads = await req.context.models.Squad.findAll({ where: { userId: req.user.id }});
+    
+    // Return No Content if the user has no squads
+    if (squads.length < 1) {
+      res.status(204);
+      return res.send({ message: "This user has no squads."})
+    }
```

Comment:
The database query and the error message are two different things here:

What you're checking is if the current user _created_ the squad, since that's all you're recording.

The error message makes it sounds like the user is not a _member_ of any squads, which isn't implemented yet.